notebooks/254-llm-chatbot/254-rag-chatbot.ipynb

```diff
@@ -0,0 +1,1286 @@
+{
```

Comment:
<p>if you rebase notebooks, probably you can see also tiny-llama-1b model. I think it is good to include it too, as it is small and fast and does not required a lot of RAM, I see you have it in widget (it is stored in your cell output), but have not in description</p><p> </p><p>For embeddings models, please provide links on some sources (sentence-transofmers, CoSENT and model cards)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1512/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>